### DISCUSSION TITLE
Doubt: React native with IOS

### DISCUSSION THREAD

**Discussion by zKriguer** [community/community#46702] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Can React Native be used to build apps for both iOS and Android platforms, while maintaining platform-specific design elements and behaviors?

**Comment by fellipeutaka** [comment, marked as answer]
Yes, React Native allows for the creation of platform-specific versions of components, allowing developers to build apps that maintain native design elements and behaviors on both iOS and Android platforms. Additionally, React Native provides platform APIs and platform-specific extensions that can be used to access platform-specific functionality and design elements 🤪
